### ISSUE TITLE
Passing individual click through links from multi-valued variable

### ISSUE THREAD

**Issue by jericvizmanos** [grafana/grafana-polystat-panel#161]
Current Issue:
When using multi valued variables with repeating panels (i.e. selecting 2 in the variable dropdown), the repeat option generates the other panels based on those selected variables. However, instead of having 2 repeated panels and passing 2 **different values** in the URL for each of those panels, it passes them together on both links.

For example, when selecting A and B in the variable dropdown, 2 panels (A and B) would get generated. However, if we try to pass a click through link containing the value of the variable selected, it would pass `{A+B}` on both of the panels instead of passing `{A}` to panel A and `{B}` to panel B.

Ask: When using multiple variables in the dropdown with repeating panels, there should be an option to pass individual variables to each respective panel.

**Comment by matletix** [comment]
I agree, and this is a **bug** (rather than an enhancement) when multiple values are selected at once in the variable drop down.
It works as expected if you use your `$variable` in the panel title for example, or in the query alias (it displays correctly as the name of the polygon if you don't use composites and overrides).
It fails by concatenating all the values at once in all the generated panels when used in fields of the grafana-polystat-panel plugin like `clickThrough`, `prefix` or `suffix`

**Comment by a-vogel-tappert** [comment]
Issue still exists. Any plans when to fix it ?

**Comment by amaspastor** [comment]
Any news on a fix for this bug ?

**Comment by briangann** [comment]
Putting this into the queue, definitely want to fix this.
